simtools/layout/layout_array.py

```diff
@@ -930,9 +937,16 @@ def include_radius_into_telescope_table(telescope_table):
         """
 
         telescope_table["radius"] = [
-            telescope_table.meta["corsika_sphere_radius"][names.get_telescope_type(tel_name_now)]
+            u.Quantity(
+                telescope_table.meta["corsika_sphere_radius"][
+                    names.get_telescope_type(tel_name_now)
+                ]
+            )
+            .to("m")
+            .value
             for tel_name_now in telescope_table["telescope_name"]
         ]
+        telescope_table["radius"] = telescope_table["radius"].quantity * u.m
```

Comment:
This is the move the unit from each entry of this table column into the column header. So instead of `"radius:" 1m, 5m, 20m` we have now `"radius | m" 1, 5, 20`, consistent with all other table columns.

Reply:
Oh, OK, didn't know it makes a difference but it's no problem of course.

Reply:
Don't know if it makes a differences, but it is consistent with all other columns in the table (where the columns have values, and not quantities)